README.md

```diff
@@ -54,3 +54,25 @@ TEMPLATES = [
     }
 ]
 ```
+
+A custom folder to load the templates can be specified. This allows you to keep the URL acessible templates separately from template includes.
+
+``` python
+# As a subfolder in templates. Only searching inside `templates/pages/`
+TEMPLATE_FINDER_DIR = 'pages'
```

Comment:
Would it be okay to change this to `TEMPLATE_FINDER_DIRECTORY`, `TEMPLATE_FINDER_PATH` or `TEMPLATE_FINDER_FOLDER` (whichever you think is more appropriate)? I'm thinking of making a set of rules in the [webteam practices](https://github.com/ubuntudesign/webteam-practices) repo including the rule that we should avoid using abbreviations or shortenings in variable names.